src/Capgemini.PowerApps.PackageDeployerTemplate/Capgemini.PowerApps.PackageDeployerTemplate.csproj

```diff
@@ -18,6 +18,9 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
I think this should go in the .editorconfig file. It looks like this only applies when building in Debug

Reply:
Ah yes!